CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Added
 
 - Support for QHD displays
+- `sandstone/PageViews` prop `fullContents` to allow contents of the page to use the entire area
```

Comment:
`entire area` may be little unclear to app developers, so let's make it clearer.

Reply:
It's fixed